mathics/builtin/graphics.py

```diff
@@ -660,32 +730,47 @@ class ColorDistance(Builtin):
 
     CIE76: euclidean distance in the LABColor space
     CIE94: euclidean distance in the LCHColor space
+    CIE2000 or CIEDE2000: CIE94 distance with corrections
+    CMC: Colour Measurement Committee metric (1984)
     DeltaL: difference in the L component of LCHColor
     DeltaC: difference in the C component of LCHColor
     DeltaH: difference in the H component of LCHColor
 
-    >> N[ColorDistance[Magenta, Green], 5]
+    It is also possible to specify a custom distance
+
+    >> ColorDistance[Magenta, Green]
      = 2.2507
+    >> ColorDistance[{Red, Blue}, {Green, Yellow}, DistanceFunction -> {"CMC", "Perceptibility"}]
+     = {1.0495, 1.27455}
```

Comment:
Looks good. 

Can we test some more cases including `CIE2000`? You can also add tests that wont show up in the docs by using `#>` instead of `>>`.